Libplanet/Tx/TxPolicyViolationException.cs

```diff
@@ -23,5 +23,21 @@ public TxPolicyViolationException(TxId txid, string message)
             : base(txid, message)
         {
         }
+
+        /// <inheritdoc cref="Libplanet.Tx.TxPolicyViolationException" />
```

Comment:
What's your intention on this?  I'm curious why it inherits docs from class `TxPolicyViolationException`.

Reply:
I was thinking that `TxPolicyViolationException` have same summary naively. 
Now think about this, It will be better if one can notice that also there's a constructor for innerException from summary.